drizzlepac/haputils/poller_utils.py

```diff
@@ -868,8 +868,9 @@ def build_poller_table(input, log_level, poller_type='svm'):
                             poller_row['skycell_obj'] = scell_obj
                             # append new row to table
                             new_poller_table.add_row(poller_row[0])
+                            poller_table = new_poller_table
```

Comment:
```suggestion
        poller_table = new_poller_table
```
This line needs to be at the same indentation level as outer most loop so that this re-assignment only occurs after all the rows have been added.